src/components/Gallery/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import AwesomeSlider from 'react-awesome-slider'
+import СoreStyles from 'react-awesome-slider/src/core/styles.scss'
+import AnimationStyles from 'react-awesome-slider/src/styled/fold-out-animation/fold-out-animation.scss'
+
+const Gallery = ({ images, ...rest }) => (
+  <AwesomeSlider
+    animation="foldOutAnimation"
+    cssModule={[СoreStyles, AnimationStyles]}
+    {...rest}
+  >
+    {images.length && images.map(image => <div data-src={image.src} />)}
+  </AwesomeSlider>
+)
+
+Gallery.propTypes = {
+  images: PropTypes.arrayOf(
+    PropTypes.shape({
+      src: PropTypes.string
+    }).isRequired
+  )
```

Comment:
Why array of objects? In director.json this`s array of strings

Reply:
I just thought it would be more flexible for future needs - with an object you are able some additional data for each image, title for example.